### ISSUE TITLE
[Docs] Merge Hello World tutorial content to Essential ECS Concepts

### ISSUE THREAD

**Issue by jeffcampbellmakesgames** [jeffcampbellmakesgames/Entitas-Redux#45]
Merge the Hello World page to [this page](https://github.com/jeffcampbellmakesgames/Entitas-Redux/wiki/Concepts)

**Comment by matthiashermsen** [comment]
@jeffcampbellmakesgames I would like to work on this :)

**Comment by matthiashermsen** [comment]
@jeffcampbellmakesgames please review this

https://github.com/jeffcampbellmakesgames/Entitas-Redux/wiki/Concepts/_compare/2566aac5186d27abae6ba885539df8e6db2fc481

I also updated the Hello World wiki page

https://github.com/jeffcampbellmakesgames/Entitas-Redux/wiki/Hello-World/_compare/1e921766213f67fdf40da2be12628e49f680d3ac

**Comment by jeffcampbellmakesgames** [comment]
After seeing the original Hello World content I had in the Concepts page, I decided I didn't like that it was mixing ECS basics with what is the manual for EntitasRedux. Instead, what I ended up doing was moving this content to the Getting Started section as it felt appropriate for a beginning user there rather than someone looking up implementation assistance in the manual.

![image](https://user-images.githubusercontent.com/1663648/104456525-cdab7580-5576-11eb-8cc7-075dfc24fe47.png)

If this looks good to you I'll mark this as done.
